src/index.html

```diff
@@ -2,12 +2,46 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+  <meta
+    name="viewport"
+    content="
+      width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0
+    "
+  >
   <title>Product cards</title>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Product cards</h1>
+  <div data-qa="card" class="card">
+    <div class="card__img"><img src="images/imac.jpeg" alt="Apple iMac 27"></div>
```

Comment:
одна строка один тег
```suggestion
    <div class="card__img">
      <img src="images/imac.jpeg" alt="Apple iMac 27">
    </div>
```